Gugudan/Gugudan/main.swift

```diff
@@ -17,3 +17,49 @@ import Foundation
  6. Functional Programming 형태로
  7. 깃허브에 올리고 코드 리뷰 요청하기
  */
+
+
+// 3. 함수로 바꿔서
+
+outputGugudanWithReadLine()
+
+// 입력받아서 출력
+func outputGugudanWithReadLine() {
+    print("몇 단을 출력할까요? (숫자만 입력해주세요)")
+    let readLine = readLine()
+    guard let column = readLine, Int(column) != nil else {
+        print("숫자만 입력해주세요.")
+        return
+    }
```

Comment:
```suggestion
    guard let column = readLine(), Int(column) != nil else {
        print("숫자만 입력해주세요.")
        return
    }
```
이렇게 인라인으로 처리해도 될 것 같네요.